vendors.d/alt_common_pes.json

```diff
@@ -0,0 +1,9 @@
+{
+    "legal_notice": "",
+    "packageinfo": [],
+    "provided_data_streams": [
+        "2.0",
```

Comment:
Just for my own education, what these version mean? Should they match with values in json mapping file - `1.1` is missing here?

Reply:
Data streams is list of X.Y strings where X is mandatory is saying with what leapp-repository packages the data is compatible, in our version we are using 2.x streams, so it's enough to have 2.x in both.